src/utils/json.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Parses the string as json. If the string is not a valid json, then an empty json is returned.
+ * @param json a json string
+ * @returns {{}}
+ */
+export function safeParseJson (json) {
+  try {
+    return JSON.parse(json)
+  } catch (e) {
+    return {}
```

Comment:
I guess we'd want to emit this error

Reply:
I removed the json parsing utility and catch all the errors that happen in the /baker calls https://github.com/LiveIntent/live-connect/pull/25/files#diff-51925b852a9fa34aa4d658fa7cd30c4dR35. The error is emitted now.